### PR TITLE
Fixed open.md

### PR DESCRIPTION
**Pull request by rselwyn**

### CODE REVIEW

File: pages/osx/open.md [hunk 1]
```diff
@@ -16,8 +16,8 @@
 
 - Reveal a file in finder
 
-`open -R /path/dir/file`
+`open -R /path/dir/{{file}}`
 
-- Open all the word docs in the current directory with Microsoft Word
+- Open all the files of that extension  in the current directory with the associated application
```
Comment (waldyrious) [inline pages/osx/open.md]: Suggestion: "files of a given extension"

**Comment by waldyrious** [comment]
Nice one! Please check the comment I left inline.

**Comment by rselwyn** [comment]
Squashed all the commits and updated!

**Comment by waldyrious** [comment]
You forgot the "in the" (current directory) :smile:

**Comment by rselwyn** [comment]
I think I got it this time :shipit: :smile:

**Comment by waldyrious** [comment]
:shipit: indeed :+1:

**Comment by igorshubovych** [comment]
@rprieto is this correct use of tokens?

**Comment by rprieto** [comment]
That looks great to me, thanks @rselwyn! Two small ones:
`. should it be `open {{*.ext}}` as well?
2. it would be nice to come up with a standard across TLDR to have a consistent approach to `{{file}}`, `{{/path/dir/file}}`, {{path_to_file}}`. Personally I'd go with `{{file}}` which is generic enough that anyone with very basic Unix knowledge knows you can specify a file name, relative path, absolute path.... or even the `<(command)` operator in bash. This goes beyond this particular page though, so I'm happy to merge as is.

**Comment by rselwyn** [comment]
@rprieto I did edit 1, but I held pack on edit 2.  I think you might want to merge these for know, and if needed, I can update it.

**Comment by waldyrious** [comment]
Looks like this one's ready to ship :)

**Pull request merged**